app/src/main/java/com/github/se/cyrcle/ui/review/ReviewScreen.kt

```diff
@@ -55,91 +57,97 @@ fun ReviewScreen(
-              verticalAlignment = Alignment.CenterVertically) {
-                for (i in 1..5) {
-                  val starIcon =
-                      when {
-                        i <= sliderValue -> Icons.Filled.Star // Full star
-                        i - 0.5f == sliderValue -> Icons.Filled.StarHalf // Half star
-                        else -> Icons.Filled.StarBorder // Empty star
-                      }
-                  Icon(
-                      imageVector = starIcon,
-                      contentDescription = null,
-                      tint = MaterialTheme.colorScheme.primary,
-                      modifier = Modifier.size(40.dp).testTag("Star$i"))
-                }
-              }
+  Scaffold(
+      topBar = {
+        TopAppBar(
+            navigationActions = navigationActions, stringResource(R.string.review_screen_title))
+      }) { paddingValues ->
+        Column(
+            modifier = Modifier.fillMaxSize().padding(paddingValues).padding(16.dp),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center) {
+              // Stars reflecting the slider value (supports half stars)
+              Row(
+                  modifier = Modifier.padding(8.dp).testTag("StarsRow"),
+                  verticalAlignment = Alignment.CenterVertically) {
+                    for (i in 1..5) {
+                      val starIcon =
+                          when {
+                            i <= sliderValue -> Icons.Filled.Star // Full star
+                            i - 0.5f == sliderValue -> Icons.Filled.StarHalf // Half star
+                            else -> Icons.Filled.StarBorder // Empty star
+                          }
+                      Icon(
+                          imageVector = starIcon,
+                          contentDescription = null,
+                          tint = MaterialTheme.colorScheme.primary,
+                          modifier = Modifier.size(40.dp).testTag("Star$i"))
+                    }
+                  }
 
-          // Display the experience text based on the slider value
-          Text(
-              text =
-                  when (sliderValue) {
-                    in 0f..1f -> "Terrible experience!"
-                    in 1f..2f -> "Bad experience :("
-                    in 2f..3f -> "Poor experience :/"
-                    3f -> "Average experience..."
-                    in 3.5f..4f -> "Good experience!"
-                    in 4.5f..5f -> "Great experience!"
-                    else -> ""
-                  },
-              style = MaterialTheme.typography.bodyLarge,
-              modifier = Modifier.padding(top = 8.dp),
-              testTag = "ExperienceText")
+              // Display the experience text based on the slider value
+              Text(
+                  text =
+                      when (sliderValue) {
+                        in 0f..1f -> ""
```

Comment:
there is a missing string here.

Reply:
There never was ^^

Reply:
I'm actually in the process of addressing this for my ReviewScreen refactor :), leave it as is for now and I'll handle it

Reply:
I had already done it by the time I read your comment, hope it will still help you somehow ^^